service_mesh/v2x/upgrading-ossm.adoc

```diff
@@ -0,0 +1,108 @@
+
+.Prerequisites
+
+* You must upgrade to {product-title} 4.6. before you upgrade to {ProductName} 2.0.
+* You must have an updated {ProductName} operator. If you selected the *automatic* upgrade path, the operator automatically downloads the latest information. However, there are steps you must take to use the features in {ProductName} to version 2.0.
+
+[id="ossm-upgrading-upgrade_{context}"]
+== Upgrading {ProductName}
+
+To upgrade {ProductName}, you must create a new project for the {ProductName} instance. Then, once it's configured, move your microservice applications and workloads to the new instance.
+
+.Procedure
+
+. Back up your control plane configuration. Switch to the project that contains your `ServiceMeshControlPlane` resource.
++
+[source,terminal]
+----
+oc project istio-system
+----
++
+.Enter the following command to retrieve the current configuration. Your <smcp_name> is specified in the metadata of your `ServiceMeshControlPlane` resource, for example `basic-install` or `full-install`.
++
+[source,terminal]
+----
+$ oc get servicemeshcontrolplanes.v1.maistra.io <smcp_name> -o yaml > <smcp_name>.v1.yaml
+----
++
+. Generate and retrieve a sample v2 `ServiceMeshControlPlane` that contains information about your configuration. You can modify this version of your `ServiceMeshControlPlane` as a starting point.
++
+[source,terminal]
+----
+$ oc get smcp <smcp_name>.v2.yaml
+----
++
+. Create a project. In the {product-title} console Project menu, click `New Project` and enter a name for your project, `istio-system-upgrade`, for example. Or, you can run this command from the CLI.
++
+[source,terminal]
+----
+$ oc new-project istio-system-upgrade
+----
++
+. Create a `ServiceMeshControlPlane` in the new namespace. On the command line, run the following command to deploy the control plane with the v2 version of the `ServiceMeshControlPlane` that you retrieved. In this example, replace `<V2_SMCP> `with the path to your file.
++
+[source,terminal]
+----
+$ oc create -n istio-system-update -f <V2_SMCP>.yaml
+----
++
+Alternatively, you can use the console to create the control plane. In the console, click *Project*. Then, select the project name you just entered.
++
+..Click *Operators* -> *Installed Operators*.
+..Click *Create ServiceMeshControlPlane*.
+..Select *YAML view* and paste the following example into the field.
++
+..Click *Create*.
+
+[id="ossm-upgrading-differences_{context}"]
+== Configuring the 2.0 `ServiceMeshControlPlane`
+
+The `ServiceMeshControlPlane` resource has been updated for {ProductName} version 2.0. Next, modify the v2 `ServiceMeshControlPlane` resource to take advantage of the new features and to fit your deployment. Consider the following changes to the specification and behavior of {ProductName} 2.0 as you're modifying your `ServiceMeshControlPlane` resource. You can also refer to the {ProductName} 2.0 product documentation for updates to features you use.
```

Comment:
Maybe one day. It's in a different PR, so it would break the build if I included it. However, I have given a set of steps that shows how to edit the control plane, so I don't think users are stranded here. The configuration of the migrated control plane is the purpose of the rest of this section and is different than the regular configuration stuff, because it contains differences with 1.1.